<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Greeter</class>
 <widget class="QWidget" name="Greeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color: white;
max-height: 10px;
}
QPushButton {
selection-background-color: rgb(255, 41, 41);
background: rgb(255, 159, 49);
}
QPushButton:hover{

background:rgb(255, 0, 0);
}
QPushButton:pressed {
background:rgb(255, 102, 0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ready to play Worms?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Choose an option:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create Match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="joinButton">
       <property name="text">
        <string>Join Match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createMapButton">
       <property name="text">
        <string>Create Map</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
